devices/wholeBodyDynamics/README.md

```diff
@@ -23,6 +23,7 @@ For an overview on `wholeBodyDynamics` and to understand how to run the device,
 | jointVelFilterCutoffInHz    | - | double            | Hz    |      -        | Yes      | Cutoff frequency of the filter used to filter joint velocities measures. | The used filter is a simple first order filter. |
 | jointAccFilterCutoffInHz    | - | double            | Hz    |      -        | Yes      | Cutoff frequency of the filter used to filter joint accelerations measures. | The used filter is a simple first order filter. |
 | startWithZeroFTSensorOffsets    | - | bool            | -    |      False       | No      | Use zero FT sensor offsets at startup. If this flag is set to false, the device estimates the offsets of FT sensors at startup||
+| disableSensorReadCheckAtStartup | - | bool            | -    |      False       | No      | If this flag is set to false, the read from the sensors is skipped at startup ||
```

Comment:
Is the description correct or it should be corrected?

Reply:
Yeah actually is the opposite